presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1325,19 +1334,14 @@ private String formatPlanNodeStatsAndCost(PlanNode node)
             PlanNodeStatsEstimate stats = lookup.getStats(node, session, types);
             PlanNodeCostEstimate cost = lookup.getCumulativeCost(node, session, types);
             return String.format("{rows: %s, bytes: %s, cpu: %s, memory: %s, network: %s}",
-                    formatEstimate(stats.getOutputRowCount()),
+                    formatAsLong(stats.getOutputRowCount()),
                     formatEstimateAsDataSize(stats.getOutputSizeInBytes()),
-                    formatEstimate(cost.getCpuCost()),
-                    formatEstimateAsDataSize(cost.getMemoryCost()),
-                    formatEstimate(cost.getNetworkCost()));
+                    formatDouble(cost.getCpuCost()),
+                    formatDouble(cost.getMemoryCost()),
+                    formatDouble(cost.getNetworkCost()));
         }
     }
 
-    private static String formatEstimate(double value)
-    {
-        return isNaN(value) ? "?" : String.valueOf(value);
-    }
-
```

Comment:
Nope, it is still used for "bytes: " formatEstimateAsDataSize(stats.getOutputSizeInBytes())